tests/run-build.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# run-build.sh
+#
+# Copyright (C) 2016 Intel Corporation
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+
+if [ "$1" == "" ]; then
+    BUILD_DIR=$(mktemp -d -p /tmp build_XXXXX)
```

Comment:
fair enough. now building in /workdir/build_XXX in the container.  cleanup on success.